spec/controllers/list_controller_spec.rb

```diff
@@ -0,0 +1,82 @@
+require 'spec_helper'
+include AdminSpecHelper
+
+RSpec.describe ListController, type: :controller do
+
+  before(:each) do
+    login_as_therapist
+    @list = FactoryGirl.create(:list, therapist: @user.role)
+  end
+  
+  describe "POST #create" do
+    it "successfully creates a list" do
+      post :create, list: {title: "A whole new world", description: "Fantastic description", therapist: @user.role}
+
+      list = JSON.parse(response.body)['list']
+
+      expect(response).to have_http_status(:success)
+      expect(list['title']).to eq("A whole new world")
+      expect(list['description']).to eq("Fantastic description")
+      expect(list['therapist_id']).to eq(@user.role.id)
+    end
+  end
+
+  describe "GET #index" do
+    it "returns http success" do
+      get :index
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:index)
+    end
+  end
+
+  describe "GET #new" do
+    it "returns http success and renders template" do
+      get :new
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:new)
+    end
+  end
+
+  describe "GET #show" do
+    it "returns http success and renders template" do
+      get :show, id: @list.id
+      expect(response).to have_http_status(:success)
+      expect(response).to render_template(:show)
```

Comment:
I haven't fleshed out tests fully for this yet since we only have a static view of requests, I may hold off on this until I pickup my next ticket addressing requests, thoughts?